dart-impl/mpi/include/dash/dart/mpi/dart_globmem_priv.h

```diff
@@ -6,6 +6,7 @@
 #define DART_GPTR_COPY(gptr_, gptrt_)                       \
   do {                                                      \
     gptr_.unitid = gptrt_.unitid;                           \
+    gptr_.flags  = gptrt_.flags;                            \
```

Comment:
1. `DART_GPTR_NULL` cannot be an inline function since it is a constant. 
2. `DART_GPTR_EQUAL` and `DART_GPTR_IS_NULL` can be.
3. `DART_GPTR_SET_FLAG` and `DART_GPTR_GET_FLAG`: They only exist in the 4.0 interface and are outdated (there is `dart_gptr_getflags` and `dart_gptr_setflags`) to handle 16 and 8 bit flags in the segment and the `gptr`, respectively. We really need to talk about the flags and how they are accessed/handled.